src/DataContainer/EntityImportSourceContainer.php

```diff
@@ -225,7 +245,13 @@ public function onLoadFileContent(?string $value, ?DataContainer $dc)
             return '';
         }
 
-        return $this->getFileContent($sourceModel);
+        try {
+            $content = $this->getFileContent($sourceModel);
+        } catch (\Exception $e) {
+            return '';
+        }
+
+        return $content;
```

Comment:
How is exception visualized to the user here?

Reply:
This is for the content field in import source in backend. It returns an empty content.